src/components/FilterForm/index.tsx

```diff
@@ -112,7 +114,7 @@ export const FilterForm: FC<TFilter> = ({
             }}
           >
             {<img src={crossIcon} alt="clear filter icon" />}
-            Clear filter
+            {t('Clear filter')}
```

Comment:
as a keys - use 'clear_filter'. (lovercase_connected_with_loverdash)
no need to fix, but write your next code according to this rule